client/app/components/Tooltip.tsx

```diff
@@ -3,7 +3,7 @@ import AntTooltip, { TooltipProps } from "antd/lib/tooltip";
 import { isNil } from "lodash";
 
 export default function Tooltip({ title, ...restProps }: TooltipProps) {
-  const liveTitle = isNil(title) ? (
+  const liveTitle = !isNil(title) ? (
```

Comment:
This feels far-reaching. Is this just a tweak to a recent change in logic? Is there any way we can add a test / screenshot or something to make sure this doesn't slip in in the future?

Reply:
It is a bug. When I added the `isNil` I accidentally made all tooltips stop working.